src/neo/Network/P2P/Payloads/ExtensiblePayload.cs

```diff
@@ -8,13 +8,39 @@
 
 namespace Neo.Network.P2P.Payloads
 {
+    /// <summary>
+    /// Represents an extensible message that can be relayed.
```

Comment:
It's not obvious what this "extensible message" is, it'd be nice to explain that (who can send it, at least).